Aptoide/appcoins-wallet-android
# [APPC-1086] change poa success notification

**What does this PR do?**

   This PR changes the current success notification so it can appear and alert the user even if the application is running in fullscreen mode

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] WalletPoAService.java

**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [APPC-1086](https://aptoide.atlassian.net/browse/APPC-1086)

  Create a new campaign and wait for all the proofs to be sent and check if a notification showed 



**Code Review Checklist**
